css/index.css

```diff
@@ -158,18 +175,137 @@ header .logo {
 }
 
 .main-content .bottom-content p {
-    padding-right: 20px;
+    /*padding-right: 20px;*/
+    padding-right: 2rem;
 }
 
 /* Contact Styles */
 
 .contact p {
-    margin: 15px 0;
+    /*margin: 15px 0;*/
+    margin: 1.5rem 0;
```

Comment:
**This applies to several areas in your code** If you can, refrain from using rem for margins. Not necessarily bad but rem is dependent on the font-size while margins are dependent on the parent it is called on.